setup.py

```diff
@@ -1,11 +1,21 @@
 #!/usr/bin/env python
 import os
 import setuptools
-
+import time
 
 os.environ.setdefault('EVENTLET_IMPORT_VERSION_ONLY', '1')
 import eventlet
 
+install_requires = [
+        'dnspython >= 1.15.0',
+        'enum34;python_version<"3.4"',
+        'greenlet >= 0.3',
+        'six >= 1.10.0',
+    ]
+if not hasattr(time, 'monotonic'):
+    install_requires.append('monotonic >= 1.4')
```

Comment:
Arguably, constant list is better.

I can imagine a bug when eventlet package/wheel is built on machine with `time.monotonic` and requirement is not added for everyone later.

Reply:
monotonic already uses builtin time when possible.

Every line of code is something that can break and something we will support later. Only benefit of optional include of monotonic is to make virtualenv few KiB smaller and install few seconds faster *only* if other packages don't depend on monotonic.

Reply:
OK, i'll set monotonic pkg for the decision .

Reply:
I've done something like this before and it won't work - if a wheel package is generated on a platform with `time.monotonic` available it won't be able to be used on platforms that lack it.